langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDBEmbeddingStoreLocalIT.java

```diff
@@ -0,0 +1,104 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.*;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.Filters;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import lombok.SneakyThrows;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+import org.bson.conversions.Bson;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+
+import java.io.File;
+import java.time.Duration;
+
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+/**
+ * If container startup timeout (because atlas cli need to download mongodb binaries, which may take a few minutes),
+ * the alternative way is running `docker compose up -d` in `src/test/resources`
+ */
+public class MongoDBEmbeddingStoreLocalIT extends EmbeddingStoreIT {
```

Comment:
this tests fails as well :(

I think it was successful when running hte first time, but then it fails every time:
`Caused by: com.mongodb.MongoCommandException: Command failed with error 8 (UnknownError): 'PlanExecutor error during aggregation :: caused by :: cannot query index while in state INITIAL_SYNC' on server localhost:58874. The full response is {"ok": 0.0, "errmsg": "PlanExecutor error during aggregation :: caused by :: cannot query index while in state INITIAL_SYNC", "code": 8, "codeName": "UnknownError", "$clusterTime": {"clusterTime": {"$timestamp": {"t": 1706194625, "i": 1}}, "signature": {"hash": {"$binary": {"base64": "Deh0AWR6o9DiYm05YILvv0KETQ0=", "subType": "00"}}, "keyId": 7328050097806114823}}, "operationTime": {"$timestamp": {"t": 1706194625, "i": 1}}}`

Reply:
Sorry about that, will fix soon

Reply:
no need to be sorry, all good :)

Reply:
This problem seems very strange, because container is isolated. I am confused with why last test affect next test.